nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-pubsub-processors/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>nifi-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-expression-language</artifactId>
```

Comment:
BOOTSTRAP_SERVERS, TOPIC, CLIENT_ID are the properties shared among Consume/ProduceKafka. If we can separate and let Consume, ProduceKafka have those as its own property, ProduceKafka to support EL but not ConsumeKafka, then the dependency can be removed.

But existing flow configuration may have to be updated if one uses EL for those properties of ConsumeKafka. 

@olegz @JPercivall How do you think?